Xamarin.Forms.Core/Stepper.cs

```diff
@@ -41,7 +43,9 @@ public class Stepper : View, IElementConfiguration<Stepper>
 				eh(stepper, new ValueChangedEventArgs((double)oldValue, (double)newValue));
 		});
 
-		public static readonly BindableProperty IncrementProperty = BindableProperty.Create("Increment", typeof(double), typeof(Stepper), 1.0);
+		public static readonly BindableProperty IncrementProperty = BindableProperty.Create(nameof(Increment), typeof(double), typeof(Stepper), 1.0);
+
+		public static readonly BindableProperty StepperPositionProperty = BindableProperty.Create(nameof(StepperPosition), typeof(int), typeof(Stepper), 0);
```

Comment:
Does this need to be a new public bindable property?

Reply:
Hi @samhouts the increment is obviously needed but the StepperPositionProperty I flip/flopped on this a few times myself.  I settled on it being a Public property in the end as it was suggested to me that someone might want an Int position of the stepper rather than having to do math using Doubles to work out where the stepper is.  Foe example they know the stepper steps at 0.125 and has 50 positions rather than work out the maths for 2/5ths they can just set it to 20 or equally read 20 and know it's 2/5th of the way.

Of course if this needs to be changed then I am happy to change after all I flip/flopped quite a few times on this.

Ta

Cliff.

Reply:
I'm not opposed to new API, but I do prefer splitting API changes into separate PRs from bug fixes whenever possible!

Reply:
@samhouts I have made the property private as suggested for this PR if it's accepted I will open another issue/PR to revert this change as I think it may be helpful in some situations.  As you say it's an API change but not a breaking change.